recipes/cargo-make/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cargo-make" %}
+{% set version = "0.37.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sagiegurari/cargo-make/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c4f36ed50ee2f6786a29c20567a70aedca5e0e101a7388f44de1ba5445f3ec3e
+
+build:
+  binary_relocation: false
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - posix  # [win]
```

Comment:
I would prefer if you simply called the commands in a bat file instead of depending on `posix` here.